schema.graphql

```diff
@@ -493,3 +493,33 @@ type StakingDailySnapshot @entity {
   totalStaked: BigDecimal!
   totalStakedUSD: BigDecimal!
 }
+
+type RewardClaim @entity {
+  " {accountID}-{tx_hash} "
+  id: ID!
+  account: Account!
+  tx_hash: String!
+
+  tokens: [RewardClaimToken!]! @derivedFrom(field: "tx_hash")
+}
+
+type RewardClaimToken @entity {
+  " {accountID}-{tx_hash}-{tokenSymbol} "
+  id: ID!
+  account: Account!
+  tx_hash: RewardClaim!
```

Comment:
RewardClaimToken is unique per tx_hash and token address. It is related to RewardClaim, so it will keep a running total across all events/markets in a given claim of the tokens awarded by that claim on that specific market.